doc-content/enterprise-only/openshift/jboss-postconfigure-proc.adoc

```diff
@@ -0,0 +1,92 @@
+** `delayedpostconfigure.sh`: An empty file, required in {PRODUCT} version {ENTERPRISE_VERSION_LONG}.
+** `add-users.cli`: The {EAP} command line interface script for configuring Elytron users or for any other CLI tasks. Add your commands between the following lines:
++
+[subs="attributes,verbatim,macros"]
+----
+embed-server --std-out=echo --server-config=standalone-openshift.xml batch
+
+  <your jboss-cli commands>
+
+run-batch quit
+----
++
+. Log in to your {OPENSHIFT} cluster with the `oc` command and change to the namespace of your deployment.
+. Create a ConfigMap with the files that you prepared by using the following command:
++
+[subs="attributes,verbatim,macros"]
+----
+oc create configmap postconfigure \
+    --from-file=add-users.cli=add-users.cli \
+    --from-file=delayedpostconfigure.sh=delayedpostconfigure.sh \
+    --from-file=postconfigure.sh=postconfigure.sh
+----
++
+ifeval::["{context}"!="openshift-operator"]
+. Add mounting of the ConfigMap to the deployment configurations for {CENTRAL} and {KIE_SERVER} by using the following commands:
++
+[subs="attributes,verbatim,macros,quotes"]
+----
+oc set volumes dc/_myapp_-kieserver --add  \
+    --configmap-name=postconfigure \
+    --mount-path=/opt/eap/extensions \
+    --default-mode=0555
+oc set volumes dc/_myapp_-{PRODUCT_INIT}centr --add  \
+    --configmap-name=postconfigure \
+    --mount-path=/opt/eap/extensions \
+    --default-mode=0555
+----
++
+Replace _myapp_ with the application name that you configured. If necessary, you can remove the commands for any deployment configurations you don't need. If you have several {KIE_SERVER} instances, you can add commands for them.
++
+The change causes redeployment of the components.
+endif::[]
+ifeval::["{context}"=="openshift-operator"]
+. Enter the following command to edit the `kieconfigs-{ENTERPRISE_VERSION_LONG}` config map:
++
+[subs="attributes,verbatim,macros,quotes"]
+----
+oc edit cm kieconfigs-{ENTERPRISE_VERSION_LONG}
+----
++
+. In the file, modify the deployment configuration under the `console:` section to add the configuration to {CENTRAL} and modify all deployment configurations under the `servers:` section to add the configuration to {KIE_SERVER} instances.
++
+In each deployment configuration, make the following changes:
++
+** Under `deploymentConfigs.metadata.spec.template.spec.containers.volumeMounts`, add the following lines:
++
+[subs="attributes,verbatim,macros,quotes"]
+----
+- name: postconfigure-mount
+  mountPath: /opt/eap/extensions
+----
++
+** Under `deploymentConfigs.metadata.spec.template.spec.containers.volumeMounts`, add the following lines:
++
+[subs="attributes,verbatim,macros,quotes"]
+----
+- name: "postconfigure-mount"
+  configMap:
+    name: "postconfigure"
+    defaultMode: 0555
+----
++
+. Save the file.
+
+.Result
```

Comment:
This is not an approved label for a procedure module. You can add the following text after the last step:

. Save the file. The new operator deployments now contain the post-configuration settings.

Then I would make the last para a note, connected to the last step:
+
NOTE: In existing....